src/Models/ExtensionWrapper.cs

```diff
@@ -89,7 +90,9 @@ public bool IsRunning()
                 return false;
             }
 
-            throw;
+            // Getting here is unexpected; log the state to handle other errors in the future.

```

Comment:
Good catch on the throw, not sure why originally it was re-throwing as the callers is not try/catch-ing it.

Nit: Not this PR, but there are a few error codes that are not unexpected here.
- RPC Server Unavailable: 0x800706BA
- RPC Call Failed: 0x800706BE
- Package Updating: 0x80073D00

https://github.com/microsoft/devhome/blob/bdfe3480a00b34b1771a2d98ab92bce6f437a2ae/services/DevHome.Services.WindowsPackageManager/Services/WinGetRecovery.cs#L21-L24

https://github.com/microsoft/devhome/blob/bdfe3480a00b34b1771a2d98ab92bce6f437a2ae/services/DevHome.Services.WindowsPackageManager/Services/WinGetRecovery.cs#L53

Reply:
I have seen crashes before of dev home randomly when updating the extension packages while Dev Home was running, so if "PackageUpdating" is one possible condition here, this would explain that crash!

Reply:
#3554